ppr-ui/src/components/mhrRegistration/HomeLocation/HomeLocationType.vue

```diff
@@ -284,12 +290,12 @@ export default defineComponent({
       localState.legalDescription = pidInfo.legalDescription
     }
 
-    const validateForms = (): void => {
+    const validateForms = async () => {
       if (props.validate) {
         // @ts-ignore - function exists
-        if (props.validate) context.refs.lotForm.validate()
+        if (localState.isLotOption) await context.refs.lotForm?.validate()
         // @ts-ignore - function exists
-        if (props.validate) context.refs.homeParkForm.validate()
+        if (localState.isHomeParkOption) await context.refs.homeParkForm?.validate()
       }
```

Comment:
I never noticed the same condition twice here! Good thing that we are removing that. 

That being said, do even care about these conditions, can we just validate both forms if `props.validate` is `true`?

Reply:
I'll do some testing... That sounds right though

Reply:
If they can bother be prompted without the conditions, you could do without the computed values as well 👍

Reply:
Good call, will do